### DISCUSSION TITLE
Adding write capability

### DISCUSSION THREAD

**Discussion by gepito** [eliben/pyelftools#502] [Ideas]
Hi,
do anyone need pyelftool with modify/write capability?
I neede it badly :) So much so I begun extending in this direction.
Any comments adn/or pointers to similar efforts are welcome!

**Comment by sevaa** [comment]
#464 
#462 
#384
#373
#355
#243
#229
#15

Some of those link to forks that support writing. Here is one:

https://github.com/mefistotelis/pyelftools

**Comment by gepito** [comment]
Hi **sevaa**,
thanks for the fast answer!
- I summarize the links above that writing elf file is not the aim of the project - which make me sad, as pyelftools has decent internal structure.
- there is a fork of this library with write capability: https://github.com/mefistotelis/pyelftools
I'm checking the extension fork...
